src/api.rs

```diff
@@ -332,6 +334,10 @@ pub mod request {
           - location: Location
           - der: Message
 
+        // request size is chosen to be smaller than the largest standard syscall, Decrypt
+        Extension:
+          - id: u8
+          - request: Bytes<{ 2 * MAX_MESSAGE_LENGTH + 2 * MAX_SHORT_DATA_LENGTH }>
```

Comment:
The idea is not to be able to serialize the `Decrypt` request into this buffer.  We have to choose some arbitrary limit for the available buffer, and the idea is to use all the memory that we already need for one of the other variants, but not more to avoid adding overhead.

Reply:
Updated the comment.